test/all_tests.dart

```diff
@@ -1,3 +1,10 @@
+// Copyright (c) 2020, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+//
+// TODO: Remove https://github.com/dart-lang/sdk/issues/43605
+// @dart=2.11
```

Comment:
At first I was worried this is a bad idea, but it's probably fine since we have `publish_to: none`...

Should we consider bumping the min SDK in the pubspec to `2.11.0-0` regardless?

Reply:
I don't think we need to, this is only in test code, and should be removed later on.